guides/authentication.md

```diff
@@ -48,14 +48,21 @@ The access token will expire after some time and result in the user get `401` fr
 
 ## How to authenticate with the SDK?
 
-There are two ways to authenticate using the SDK:
+SDK supports few authentication flows:
+ - [CDF auth flow](#cdf-auth-flow)
+ - [Azure AD auth flow](#azure-ad-auth-flow)
+
+## CDF auth flow
+
+There are two ways to authenticate using the CDF auth flow:
 
 - [With redirect](#authentication-with-redirects)
 - [With popups](#authentication-with-popups)
 
 ### Authentication with redirects
 
-When doing authentication with redirects the currect browser window will be redirected to the identity provider for the user to log in. After login the same browser window will be redirect back to your application.
+When doing authentication with redirects the current browser window will be redirected to the identity provider for the user to log in.
+After the login, the same browser window will be redirected back to your application.
```

Comment:
After signing in, the browser window will be redirected back to your application.